docs/src/main/sphinx/connector/hive.rst

```diff
@@ -456,6 +456,18 @@ with Parquet files performed by the Hive connector.
         ``parquet_use_column_names``.
       - ``true``
 
+OpenX JSON format support
```

Comment:
Let’s remove this section entirely. The connector already supports JSON, so it’s confusing. We expect that it would handle existing serde properties. That’s basically saying “it works”.

If we really care to mention this serde, we could add a note at the top near the list of supported formats.